### PR TITLE
Makefile: highlight board acpi file warning msg

### PR DESCRIPTION
**Pull request by jsun26intel**
Move the warning message to the end of build process to hint user to check
the validity of $(BOARD)_acpi_info.h, otherwise user might not be aware the
existence of this file and then cause issues.

Another reason is when user uses make menuconfig method to make hypervisor
with XML files, we should give this warning message also.

Tracked-On: #3779

Signed-off-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: Makefile [hunk 1]
```diff
@@ -133,7 +127,9 @@ hypervisor:
 		$(MAKE) -C $(T)/misc/efi-stub HV_OBJDIR=$(HV_OUT) EFI_OBJDIR=$(HV_OUT)/$(EFI_OUT); \
 	fi
 #endif
-
+	@if [ -f $(GENED_ACPI_INFO_HEADER) ] && [ "$(CONFIG_XML_ENABLED)" != "true" ]; then \
```
Comment (binbinwu1) [inline Makefile]: I am not sure if this is a noticeable hint for users, better than the previous version though.
If the user make hypervisor and device model together, it is also easy to be ignored.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by jsun26intel** [comment]
> LGTM, could you help to add the description of the added log file and tell the users to check the warnings to somewhere in the doc?

sure, I will add this on doc later.

**Pull request merged**